boards/xtensa/lilygo_t3/lilygo_t3.yaml

```diff
@@ -0,0 +1,18 @@
+identifier: lilygo_t3
+name: Lilygo T3
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+supported:
+  - gpio
+  - i2c
+  - spi
+  - watchdog
+  - uart
+  - pinmux
```

Comment:
display and lora gone again?
and how about nvs?

Reply:
about the nvs, I'm unsure if that is supposed to be something like an I2C eeprom or internal flash is also considered nvs.

Reply:
all other esp32 boards have nvs :) and yeah, internal flash counts as nvs